pybossa/view/projects.py

```diff
@@ -3845,10 +3845,6 @@ def assign_users(short_name):
     ensure_authorized_to('read', project)
     ensure_authorized_to('update', project)
     access_levels = project.info.get('data_access', None)
-    if not data_access_levels or not access_levels:
-        flash('Cannot assign users to a project without data access levels', 'warning')
-        return redirect_content_type(
-            url_for('.settings', short_name=project.short_name))
 
```

Comment:
On private instance, will removing this code allow users w/o data access to be assigned to project?

Reply:
We have another check for users w/o data access: https://github.com/bloomberg/pybossa/blob/215927af7ddbd0ad04fda65468150d55b83cd93c/pybossa/view/projects.py#L3849

So they would not be allowed